### DISCUSSION TITLE
Gatsby Remark using GatsbyLink

### DISCUSSION THREAD

**Discussion by tnorlund** [gatsbyjs/gatsby#30400] [Help] [answered]
I'm currently using '.mdx' files to organize my blog. I'd like to link between them using `[]()` format. When I click on the link, it redirects me to page, rather than using GatsbyLink. Is there a way to use GatsbyLink in '.mdx' files?

**Comment by LekoArts** [comment, marked as answer]
You can use https://www.gatsbyjs.com/plugins/gatsby-plugin-catch-links/
